scripts/deploy.ts

```diff
@@ -33,6 +33,7 @@ async function main() {
     'phase2-assets/0_setup_deployments.ts',
     'phase2-assets/1_deploy_assets.ts',
     'phase2-assets/2_deploy_collateral.ts',
+    'phase2-assets/3_deploy_fToken_collateral.ts',
```

Comment:
for the reth pr i put a folder `collaterals` under `phase2-assets` and just ignored the leading numbers on the filenames since they seem arbitrary as we add more collateral https://github.com/reserve-protocol/protocol/pull/691/files#diff-07d0660c0323513348cb478ac8d90b57cb1b1605dc259aba0fd5bb830a43ab1d

Reply:
Nice, I'll mimic that here. Getting bitten a bit by the fact that we have so many active branches/PRs open